pages/open_cas_linux/guide_monitoring_json.md

```diff
@@ -0,0 +1,497 @@
+---
+title: Open CAS Linux - Admin Guide
+last_updated: October 15, 2021
+toc: true
+permalink: guide_monitoring_json.html
+---
+
+Monitoring Open CAS Linux - JSON API
+=====================
+
+# Overview
+
+Json API provides basic functionalities for inspecting CAS. Retrieving CAS statistics on various levels, listing CAS devices and providing cache, core or io class details. All requests, responses and errors are in json format.
+
+**Json request**
+
+Request consist of header with name of command and parameters with additional request detials as cache id, core id or io class id. Different parameters are required for various commands.
+
+**Json responses**
+
+Responses consists of all requested components or in case of trouble error also encoded in json format.
+
+# Statistics
+
+Statistics can be requested on different levels cache, core or io class. There are four specyfic command with different parameters responsible for inspecting those statistics.
+
+**Statistics requests:**
+
+-   core with specyfic io class statistics
+
+-   Cache statistics
+
+-   Core statistics
+
+-   io class statistics
+
+# Example statistics requests
+
+**Request of statistics on specyfied cache**
+
+```json
+{
+  "header": {
```

Comment:
Wouldn't it be better to remove _header_ group and put _command_ at the top (at the same level as _params_)?